src/test/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/HaplotypeCallerIntegrationTest.java

```diff
@@ -71,6 +73,30 @@ public void testVCFModeIsConsistentWithPastResults(final String inputFileName, f
         IntegrationTestSpec.assertEqualTextFiles(output, expected);
     }
 
+    @Test
+    public void testSitesOnlyMode() {
+        File out = createTempFile("GTStrippedOutput", "vcf");
+        final String[] args = {
+                "-I", NA12878_20_21_WGS_bam,
+                "-R", b37_reference_20_21,
+                "-L", "20:10000000-10100000",
```

Comment:
This seems like an unnecessarily large interval for this test. How about 10k bases instead of 100k?

Reply:
done